Gems/Atom/RHI/Code/Source/RHI/BufferView.cpp

```diff
@@ -75,4 +75,18 @@
             m_buffer = nullptr;
         }
     }
- }
\ No newline at end of file
+    
+    uint32_t BufferView::GetBindlessIndices(int deviceIndex, uint32_t* outReadWriteIndex) const
+    {
+        const auto& deviceBufferView = GetDeviceBufferView(deviceIndex);
+        if (!deviceBufferView)
+        {
+            return DeviceBufferView::InvalidBindlessIndex;
+        }
+        if (outReadWriteIndex != nullptr)
+        {
+            *outReadWriteIndex = deviceBufferView->GetBindlessReadWriteIndex();
+        }
+        return deviceBufferView->GetBindlessReadIndex();
+    }
```

Comment:
Can we change this function signature to be:
void BufferView::GetBindlessIndices(int deviceIndex, uint32_t* outReadWriteIndex, uint32_t* outReadIndex)
This would more clearly communicate the intent of how the function should be called. Right now having ReadIndex be the return value of the function while ReadWriteIndex gets passed out through a pointer is confusing.

Reply:
That was my original design ;-). I'll change it to its original form. I had doubts which version makes more sense, because the original version would allow `nullptr, nullptr` and behave like a useless no-op.